src/Core/src/Hosting/IMauiHandlersServiceProvider.cs

```diff
@@ -8,9 +8,9 @@ public interface IMauiHandlersServiceProvider : IMauiServiceProvider
 	{
 		Type? GetHandlerType(Type iview);
 
-		IViewHandler? GetHandler(Type type);
+		IElementHandler? GetHandler(Type type);
 
-		IViewHandler? GetHandler<T>() where T : IView;
+		IElementHandler? GetHandler<T>() where T : IElement;
```

Comment:
This is a "breaking" change, but is required to get handlers for things that are not views - like windows and maybe menu items.